widget.css

```diff
@@ -28,8 +28,8 @@ a:hover {
 }
 
 .main-container {
+  border-bottom: 2px solid #858585;
```

Comment:
I would like to not include the border in the widget because it can then be used as an inline element as well (not full width / somewhere else than at the top).

The border in the design illustrated that Enrique would add it on CFY. We can still set that custom style on our site